tsdb/head.go

```diff
@@ -2345,3 +2345,13 @@ type noopSeriesLifecycleCallback struct{}
 func (noopSeriesLifecycleCallback) PreCreation(labels.Labels) error { return nil }
 func (noopSeriesLifecycleCallback) PostCreation(labels.Labels)      {}
 func (noopSeriesLifecycleCallback) PostDeletion(...labels.Labels)   {}
+
+func (h *Head) Size() int64 {
+	walSize, _ := h.wal.Size()
```

Comment:
@codesome should we check for nil wal? here?

Reply:
A `nil` check should be fine, lets add it, though we always use WAL with Prometheus (can be nil only for tests or if external user is using without a WAL).

Reply:
Made some changes here, I'm not entirely sure if that is what you wanted